OxfordBodleian/Juel-Jensen/BDLaethg32.xml

```diff
@@ -87,21 +236,119 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <q xml:lang="am"><persName ref="PRS13882Kasahun"><roleName type="title">አለቃ</roleName> ካሳሁን</persName> <gap reason="illegible"/></q>
+                                </item>
+                                
+                                <item xml:id="a6">
+                                    <locus target="#6v"/>
+                                    <desc type="Unclear">Note in the upper margin. Only the beginning is legible.</desc>
+                                    <q xml:lang="am">ከመፍትሔ፡ ሥር፡</q>
+                                </item>
+                                
+                                <item xml:id="a7">
+                                    <locus target="#9r"/>
+                                    <desc type="MagicText">Protective prayer in Gǝʿǝz and Amharic, probably by the same
+                                        hand as <ref target="#a2"/>.</desc>
+                                    <q xml:lang="gez">ፈቲነ፡ ብይርዕ፡ ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ ይስራኤል፡ ድፍን፡ 
+                                    <gap reason="illegible" unit="chars" quantity="1"/>የል፡ ዘደፈንኮ፡ ልቦ፡ ለዲያብሎስ፡ ዘበውቄያኖለጽ፡
+                                        <gap reason="illegible" unit="chars" quantity="1"/>ትዕ፡ ከመ፡ እብነ፡ አቤል፡ ጽኑ<gap reason="illegible" unit="chars" quantity="1"/>
+                                        ከመ፡ አድማስ፡ ጽኑእ፡ ከመ፡ <gap reason="illegible" unit="chars" quantity="1"/>ብነ፡ ቤልሆር፡ ጽኑእ፡ ከማሁ፡
+                                        ድፍን፡ ልቦሙ፡ ወአድክም፡ ኃይሎሙ፡ አይናትሁ፡ <gap reason="illegible" unit="chars" quantity="3"/>
+                                        ጨለማ፡ ጉልበታችሁ፡ <gap reason="illegible" unit="chars" quantity="1"/>ዲሁን፡ ቄጠማ፡ የኑ፡ 
+                                        <gap reason="illegible"/>ቶች፡ የማውቃችሁ፡ <gap reason="illegible"/>ውቃችሁን፡ <gap reason="illegible"/></q>
+                                </item>
+                                
+                                <item xml:id="e1">
+                                    <locus target="#8v"/>
+                                    <desc>A protective square of four lines with four characters per line follows the end of the main text.</desc>
+                                    <q xml:lang="gez"/>
+                                </item>
+                                
+                                <item xml:id="e2">
+                                    <locus target="#8v"/>
+                                    <desc>Scribbles in the lower margin.</desc>
+                                </item>
+                                
+                                <item xml:id="e3">
+                                    <desc>Scribal corrections are very rare. On <locus target="#6r #8r"/>, words are marked for deletion by
+                                    expunctuation or strikethrough. Spaces left for insertion of the owner's name have
```

Comment:
Again, "expunctuation".

Reply:
How would you express it?

Reply:
"encircling" or "over- and underlining", I think :)